unittests/AST/ASTImporterTest.cpp

```diff
@@ -2254,129 +2254,229 @@ TEST_P(ImportFunctions,
             }).match(ToTU, functionDecl()));
 }
 
+//FIXME Move these tests to a separate test file.
+namespace TypeAndValueParameterizedTests {
+
+// Type parameters for type-parameterized test fixtures.
+struct GetFunPattern {
+  using DeclTy = FunctionDecl;
+  BindableMatcher<Decl> operator()() { return functionDecl(hasName("f")); }
+};
+struct GetVarPattern {
+  using DeclTy = VarDecl;
+  BindableMatcher<Decl> operator()() { return varDecl(hasName("v")); }
+};
+
+// Values for the value-parameterized test fixtures.
+// FunctionDecl:
+auto *ExternF = "void f();";
+auto *StaticF = "static void f();";
+auto *AnonF = "namespace { void f(); }";
+// VarDecl:
+auto *ExternV = "extern int v;";
+auto *StaticV = "static int v;";
+auto *AnonV = "namespace { extern int v; }";
+
 // First value in tuple: Compile options.
 // Second value in tuple: Source code to be used in the test.
-class ImportFunctionsVisibilityChain
-    : public ASTImporterTestBase,
-      public ::testing::WithParamInterface<
-          std::tuple<ArgVector, const char *>> {
+using ImportVisibilityChainParams = ::testing::WithParamInterface<
+          std::tuple<ArgVector, const char *>>;
```

Comment:
Was this indentation result of automatic formatting (looks a bit "odd")? Maybe it was left from the previous code but it should be 4 spaces now.